runtime/src/iree/task/api.c

```diff
@@ -107,6 +107,23 @@ IREE_FLAG(string, task_topology_performance_level, "any",
           "Selects only cores that match the specified performance level from\n"
           "[`any`, `low` (or `efficiency`), `high` (or `performance`)].");
 
+IREE_FLAG(
+    string, task_topology_distribution, "scatter",
+    "Strategy for distributing cores across cache domains (CCXs) within the\n"
+    "selected NUMA node(s) (use --task_topology_nodes to control NUMA "
+    "locality):\n"
+    "  `compact` - Fill cache domains sequentially (better for compute).\n"
+    "  `scatter` - Round-robin across domains (better for memory bandwidth).");
+
+IREE_FLAG(
+    string, task_topology_favor, "",
```

Comment:
I had difficulty parsing this because as `favor` is a suffix here, I tried parsing it as a noun instead of a verb. Then I mentally parsed it as `flavor`.

Replace `favor` by another term that is unambiguously either a verb ? E.g. `prefer` or `optimize` ?